From fd99bd85f5820cb6ef962e028684c3ffdcb46d16 Mon Sep 17 00:00:00 2001
From: Himanshu <himanshu@cloudera.com>
Date: Sun, 14 Oct 2012 17:44:51 -0600
Subject: [PATCH 011/202] CLOUDERA-BUILD. Upgrade Jetty to 6.1.26.cloudera.2.

This version of Jetty has been found to be much more
reliable in the context of MapReduce, so should be used
in HBase as well.

The source for this Jetty release can be found at
https://github.com/cloudera/jetty-hadoop-fix and corresponds to
commit 51042f4 in that repository.

Author: Todd Lipcon
Ref: CDH-7767
---
 pom.xml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/pom.xml b/pom.xml
index dd1b0e2..e47cca4 100644
--- a/pom.xml
+++ b/pom.xml
@@ -985,7 +985,7 @@
     <jackson.version>1.8.8</jackson.version>
     <jasper.version>5.5.23</jasper.version>
     <jaxb-api.version>2.1</jaxb-api.version>
-    <jetty.version>6.1.26</jetty.version>
+    <jetty.version>6.1.26.cloudera.2</jetty.version>
     <jetty.jspapi.version>6.1.14</jetty.jspapi.version>
     <jersey.version>1.8</jersey.version>
     <jruby.version>1.6.5</jruby.version>
-- 
1.7.0.4

